guarddog/__init__.py

```diff
@@ -1 +1 @@
-from guarddog.scanners.package_scanner import PackageScanner  # NOQA
+from guarddog.scanners.pypi_package_scanner import PypiPackageScanner  # NOQA
```

Comment:
Missing the NPM one. I'm guessing the programmatic usage for NPM doesn't work as is

As a side note, would be great to have integration tests for programmatic usage (created #110 to track it)

Reply:
+1